ksqldb-execution/src/main/java/io/confluent/ksql/schema/query/QuerySchemas.java

```diff
@@ -129,24 +144,35 @@ public SchemaInfo getTopicInfo(final String topicName) {
 
     final Set<String> keyLoggerNames = kvLoggerNames.getOrDefault(true, ImmutableSet.of());
     if (keyLoggerNames.size() != 1) {
-      throw new IllegalStateException("Multiple key logger names registered for topic."
+      final String result = keyLoggerNames.size() == 0 ? "Zero" : "Multiple";
+
+      throw new IllegalStateException(result + " key logger names registered for topic."
           + System.lineSeparator()
           + "topic: " + topicName
           + "loggers: " + keyLoggerNames
       );
     }
 
     final Set<String> valueTopicNames = kvLoggerNames.getOrDefault(false, ImmutableSet.of());
-    if (valueTopicNames.size() != 1) {
-      throw new IllegalStateException("Multiple value logger names registered for topic."
+    if (valueTopicNames.size() > 1) {
+      LOG.debug("Multiple value logger names registered for topic."
```

Comment:
I changed this code. If the multiple values do not have the same value format, then it will throw an exception. I left a comment about the zero cases too. I cannot throw with zero cases.